TestAssets/TestProjects/FilterProfiles/NewtonsoftMultipleVersions.xml

```diff
@@ -0,0 +1,4 @@
+<CacheArtifacts>
+  <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+  <PackageReference Include="Newtonsoft.Json" Version ="9.0.2-beta2"/>
```

Comment:
We talked offline, the problem with using any msbuild conventions is that it suggests other msbuild idioms might be possible. e.g. Can I use Remove? Exclude? Put common versions in props? etc.